js/pages/concept-sets/conceptset-manager.js

```diff
@@ -266,7 +266,8 @@ define([
 				} finally {
 				}
 			}));  
-			
+
+			this.conceptSetStore.resolveConceptSetExpression();
```

Comment:
resolveConceptSetExpression is highly expensive operation. It could be dozens of concepts inside a conceptSet.
The idea was to load included concepts only when `Included` tab is explicitly selected or conceptSet was modified.

Reply:
moved this call to included concepts tab

Reply:
so better

Reply:
I think this has lead to the issue reported in https://github.com/OHDSI/Atlas/issues/2421.  Previously, there was an 'await resolveConceptSetExpression()' which did hold the UI up until the initial conceptIDs for the concept set were loaded, and it's probably the case that this did not need to be an await (ie: load asynchronously) but I also thought there was a reason I did this (I can't remember now).

Reply:
Also, just to note: resolveConceptSetExpression just load the set of included conceptIDs, not the full-blown concept objects (with names/domains, etc) which is a more expensive operation.  You are correct that resolveConceptSetExpression is an expensive operation, but it's only half the work to actually load the 'includedConcepts' (which coverts conceptIDs -> Concept objects)